CONTRIBUTING.md

```diff
@@ -1,3 +1,37 @@
+# CONTRIBUTING.md
```

Comment:
We can strike this. We know what file we're in :-)

Or, alternately, just maybe make it 
`# Contributing` ?

Reply:
I'm going to make it `Contributing to the TTS Handbook` to match the `README.md` header